docs/application/dotnet/guides/uix/sticker.md

```diff
@@ -0,0 +1,353 @@
+
+- [Inserting the sticker information](#insert_sticker)
+
+  - You can store the sticker information using the `Tizen.Uix.Sticker.StickerProvider` class at application runtime, and [use the JSON file](#set_json) to store the sticker information at a time.
+  - You can store sticker information automatically when the Sticker provider application is installed without using the `Tizen.Uix.Sticker` methods.
+
+- Retrieving the sticker information
+
+  You can [retrieve the sticker information](#retrieve_sticker), which includes URI, keyword, group name, and so on.
+
+**Figure: Sticker Mechanism**
+
+![Sticker Mechanism](./media/sticker.png)
+
+## Prerequisites
+
+To enable your Sticker provider application to use the Sticker functionality:
+
+1. To use the `Tizen.Uix.Sticker.StickerProvider` class, the following category must be added to the `tizen-manifest.xml` file:
+
+    ```
+    <category name="http://tizen.org/category/sticker"/>
+    ```
+
+2. To use the methods and properties of the Tizen.Uix.Sticker namespace, include it in your application:
+
+    ```csharp
+    using Tizen.Uix.Sticker;
+    ```
+
+3. Initialize the Sticker provider with the `Initialize()` method of the `Tizen.Uix.Sticker.StickerProvider` class:
+
+    ```csharp
+    void Initialize()
+    {
+        StickerProvider.Initialize();
+    }
+    ```
+
+4. When the Sticker provider is no longer needed, deinitialize it:
+
+    ```csharp
+    void Deinitialize()
+    {
+        StickerProvider.Deinitialize();
+    }
+    ```
+
+To enable your Sticker consumer application to use the Sticker functionality:
+
+1. To use the `Tizen.Uix.Sticker.StickerConsumer` class, the application has to request permission by adding the following privilege to the `tizen-manifest.xml` file:
+
+    ```
+    <privileges>
+       <privilege>http://tizen.org/privilege/mediastorage</privilege>
+    </privileges>
+    ```
+
+2.  To use the methods and properties of the Tizen.Uix.Sticker namespace, include it in your application:
+
+    ```csharp
+    using Tizen.Uix.Sticker;
+    ```
+
+3. Initialize the Sticker consumer with the `Initialize()` method of the `Tizen.Uix.Sticker.StickerConsumer` class:
+
+    ```csharp
+    void Initialize()
+    {
+        StickerConsumer.Initialize();
+    }
+    ```
+
+4. When the Sticker consumer is no longer needed, deinitialize it:
+
+    ```csharp
+    void Deinitialize()
+    {
+        StickerConsumer.Deinitialize();
+    }
+    ```
+
+
+<a name="provider_application"></a>
+## Create Sticker Provider Application
+
+You can provide the sticker information to applications that use stickers. If you store the stickers into the Sticker database, the Sticker consumer applications can search for them.
+
+<a name="set_sticker_data"></a>
+### Set Sticker Information to StickerData class
+
+To store the sticker information to the Sticker database without using the JSON file, the sticker information must be set into the `Tizen.Uix.Sticker.StickerData` class:
+
+    ```csharp
```

Comment:
Hello @mijin85cho,
This seems very odd. The code tags on the markdown file have been declared properly. However, could you suggest any other reason for this issue?

Reply:
Hello, @omar-srbd 
It's because of the indentation.

4 spaces indentations are needed when it belongs to a list item.
So line 90 is working well. 
(Line 90 belongs to line 88, the list item, "4. When the Sticker...").

However, even though line 106 is not a list item, there are 4 spaces indentation before line 108.
So the code block is broken.

Would you please remove the indentation in the case the code block is not under a list item?
Same thing is happening to line 143, etc.

Reply:
Thank you for your detailed explanation of the issue.
Should I open another PR with the file `docs/application/dotnet/guides/uix/sticker.md` and implement changes to fix the code blocks?
Please let me know.

Reply:
As the PR was merged, to update the content, yes, another PR is needed. Would you please open another PR to proceed it?

Reply:
Sure, I will open the new PR right away.
Thank you.

Reply:
@mijin85cho,
The changes on the code blocks have been implemented on https://github.com/Samsung/tizen-docs/pull/1720 as per your instruction.
Would you please check if everything is perfectly in place as it should be?

Reply:
I checked that in PR 1720 the issue fixed.

![image](https://user-images.githubusercontent.com/9958374/198209393-569d785b-8ba2-4b78-8e05-cb0df6c3a7c5.png)